variables.tf

```diff
@@ -29,3 +29,9 @@ variable "publish" {
   type        = bool
   default     = false
 }
+
+variable "timeout" {
+  description = "Lambda Function timeout in seconds"
+  type        = string
```

Comment:
Should the type be `number`? I looked at https://github.com/terraform-aws-modules/terraform-aws-lambda for comparison.

Reply:
It should. Good catch